test/000_fullchain-boost.test.ts

```diff
@@ -71,7 +72,10 @@ async function extractRegistryEntryAddress(
 ): Promise<string> {
     const events = extractEventsFromReceipt(receipt, registryInstanceAddress, 'Transfer');
     if (events && events[0].args) {
-        const lowercaseAddress = events[0].args['tokenId'].toHexString();
+        const lowercaseAddress = ethers.utils.hexZeroPad(
+            BigNumber.from(events[0].args['tokenId']).toHexString(),
+            20,
+        );
```

Comment:
Fix for:
`Error: invalid address (argument="address", value="0x7a96932ea86a8a23f336f4800f404e6361d42b", code=INVALID_ARGUMENT, version=address/5.7.0)` occurring some times.